content/chainguard/libraries/java/build-configuration.md

```diff
@@ -255,3 +283,9 @@ These tools also include their own mechanisms to configure repositories for
 binary artifact retrieval. Consult the specific documentation and adjust your
 configuration to use your repository manager and newly created repository group
 or virtual repository.
+
+Example URLs for repository managers:
```

Comment:
This seems like it shouldn't be here, as the same content is added above and I'm not sure it fits here.

Reply:
Users of Gradle and Maven are very separate so I think we need to repeat ..